docs/project/dogfooding.md

```diff
@@ -185,181 +185,181 @@ $ bin\Debug\net6.0\win-x64\publish\App.exe
-[linux-musl-arm-version-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/productVersion.txt
-[linux-musl-arm-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-linux-musl-arm.tar.gz
-[linux-musl-arm-targz-checksum-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-linux-musl-arm.tar.gz.sha512
-[linux-musl-arm-nethost-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-nethost-linux-musl-arm.tar.gz
-[linux-musl-arm-symbols-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-symbols-linux-musl-arm.tar.gz
-
-[linux-musl-arm64-badge-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/sharedfx_linux-musl-arm64_Release_version_badge.svg?no-cache
-[linux-musl-arm64-version-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/productVersion.txt
-[linux-musl-arm64-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-linux-musl-arm64.tar.gz
-[linux-musl-arm64-targz-checksum-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-linux-musl-arm64.tar.gz.sha512
-[linux-musl-arm64-nethost-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-nethost-linux-musl-arm64.tar.gz
-[linux-musl-arm64-symbols-targz-6.0.X]: https://aka.ms/dotnet/net6/dev/Runtime/dotnet-runtime-symbols-linux-musl-arm64.tar.gz
+[win-x64-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/sharedfx_win-x64_Release_version_badge.svg?no-cache
+[win-x64-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/runtime-productVersion.txt
+[win-x64-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x64.exe
+[win-x64-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x64.exe.sha512
+[win-x64-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x64.zip
+[win-x64-zip-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x64.zip.sha512
+[win-x64-nethost-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-nethost-win-x64.zip
+[win-x64-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-symbols-win-x64.zip
+
+[win-x86-badge-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/sharedfx_win-x86_Release_version_badge.svg?no-cache
+[win-x86-version-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/runtime-productVersion.txt
+[win-x86-installer-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.exe
+[win-x86-installer-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.exe.sha512
+[win-x86-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.zip
+[win-x86-zip-checksum-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-win-x86.zip.sha512
+[win-x86-nethost-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-nethost-win-x86.zip
+[win-x86-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtimedotnet-runtime-symbols-win-x86.zip
```

Comment:
this link does not work either.

Reply:
ah, 
```suggestion
[win-x86-symbols-zip-6.0.X]: https://aka.ms/dotnet/6.0/daily/Runtime/dotnet-runtime-symbols-win-x86.zip
```

Reply:
hmm, that doesn't seem to work either.

You might consider just checking all your links with a quick script and curl.

Reply:
I wonder if that one even exists anymore. I don't see it being published.

Reply:
The last version of that file that was published was the 6.0.0-alpha.1.20562.6 version. Are you ok with me removing links in the table that are no longer being published?